integrationTests/ts/__tests__/integration.test.ts

```diff
@@ -336,36 +336,32 @@ describe("Integration tests", function test() {
       }
 
       // prove on chain if everything matches
-      await expect(
-        proveOnChain({
-          pollId,
-          proofDir: path.resolve(__dirname, "../../../cli/proofs"),
-          subsidyEnabled,
-          maciAddress: contracts.maciAddress,
-          messageProcessorAddress: pollContracts.messageProcessor,
-          tallyAddress: pollContracts.tally,
-          subsidyAddress: pollContracts.subsidy,
-          signer,
-        }),
-      ).to.eventually.not.rejectedWith();
+      await proveOnChain({
+        pollId,
+        proofDir: path.resolve(__dirname, "../../../cli/proofs"),
+        subsidyEnabled,
+        maciAddress: contracts.maciAddress,
+        messageProcessorAddress: pollContracts.messageProcessor,
+        tallyAddress: pollContracts.tally,
+        subsidyAddress: pollContracts.subsidy,
+        signer,
+      }).catch((err) => expect(err).to.eq(undefined));
```

Comment:
If the error is not thrown this catch won't be fired but in case if there is an error it will fail because error is not undefined.
Is it possible to check return value or if it resolves properly?

Reply:
maybe we don't even add except and just let it throw if it throws? so the test fails?

Reply:
none of those functions return anything, and the matcher we used before seem to not work. is there any other matcher you know of that could work?

Reply:
Usually, it's a good practice to have expect for resolved values as well, because it can be resolved with undefined or null values, but it will be considered as valid tests.

Reply:
I added .to.not.be.rejected and seems to work fine